public/content/documentation/native/controls/table-row-button.md

```diff
@@ -0,0 +1,302 @@
+## General Notes
+
+How to test a table row button / list item
```

Comment:
so we don't add "button" as in "table row button" in current MagentaA11y. Just noting in case this wasn't intentional.

Reply:
oh, so I see it in the OG site within the URL but didn't see it in the "how to test a ..."

Up to you.

Reply:
I went for "Table row button" because of the URL (https://www.magentaa11y.com/checklist-native/table-row-button/) as well as the "Role: Identifies as a button in iOS and "double tap to activate" in Android" in condensed criteria (which I've referred to it as such with native app teams).